src/Microsoft.VisualBasic/tests/ConversionsTests.cs

```diff
@@ -2184,33 +2357,39 @@ public static IEnumerable<object[]> ToBoolean_Object_TestData()
         [Theory]
         [MemberData(nameof(ToBoolean_Object_TestData))]
         [MemberData(nameof(ToBoolean_String_TestData))]
-        public void ToBoolean_Object_ReturnsExpected(object value, bool expected)
+        public void ToBoolean_Object_ReturnsExpected(IConvertible value, bool expected)
         {
             AssertEqual(expected, Conversions.ToBoolean(value));
+            if (value != null)
+            {
+                AssertEqual(expected, Conversions.ToBoolean(new ConvertibleWrapper(value)));
+            }
         }
 
         public static IEnumerable<object[]> ToBoolean_InvalidObject_TestData()
         {
-            yield return new object[] { char.MinValue };
-            yield return new object[] { (char)1 };
-            yield return new object[] { char.MaxValue };
-            yield return new object[] { new DateTime(10) };
```

Comment:
Are these cases no longer relevant?

Reply:
I think they’re consolidated with the test data for other methods as they’re common with all other conversion methods